ingestion/src/metadata/profiler/processor/core.py

```diff
@@ -145,31 +145,31 @@ def columns(self) -> List[Column]:
         by skipping the columns to ignore.
         """
 
-        if self._columns:
-            return self._columns
+        if self.column_list:
+            return self.column_list
```

Comment:
why do you modify the interface? If you need to set it define a setter for the property

Reply:
Yes, Let me revert that change

Reply:
modified, Thanks